### PR TITLE
Include client id in email and sms template data

### PR DESCRIPTION
**Pull request by IniZio**
ref: #2901

### CODE REVIEW

File: pkg/lib/translation/service.go [hunk 1]
```diff
@@ -119,17 +120,20 @@ func (s *Service) emailMessageHeader(name string, args interface{}) (sender, rep
 }
 
 func (s *Service) EmailMessageData(msg *MessageSpec, args interface{}) (*EmailMessageData, error) {
-	sender, replyTo, subject, err := s.emailMessageHeader(msg.Name, args)
+	data := map[string]interface{}{"ClientID": clientid.GetClientID(s.Context)}
```
Comment (carmenlau) [inline pkg/lib/translation/service.go:123]: The translation service should handle constructing the message data only, it should not have context about client id. The client id should be provided by the caller. Thought?

Comment (carmenlau) [reply pkg/lib/translation/service.go:123]: Any updates?

Comment (IniZio) [reply pkg/lib/translation/service.go:123]: @carmenlau If so then I think `ClientID` should be only included for magic link generate code email? Kind of weird to enforce client ID passing for messages that should not need client ID at all.

Comment (IniZio) [reply pkg/lib/translation/service.go]: Updated thanks!

Comment (louischan-oursky) [reply pkg/lib/translation/service.go]: We need to have ClientID for all outgoing messages. I think EmailMessageData is the safest place to handle this. ClientID will never be missed. Please revert back to the original implementation.

Comment (IniZio) [reply pkg/lib/translation/service.go:123]: Updated :pray:

**Pull request merged**